### PR TITLE
changes

### PR DESCRIPTION
**Pull request by brachiWeil**
first2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,5 +9,123 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (brachiWeil) [inline index.html:31]: t45r

Comment (brachiWeil) [reply index.html:31]: y

File: index.html [hunk 2]
```diff
@@ -9,5 +9,123 @@
...
```
Comment (brachiWeil) [inline index.html:62]: b

Comment (BatyaKastner0) [reply index.html:62]: brachi succedd

Comment (brachiWeil) [reply index.html:62]: batyaddddddddd

Comment (BatyaKastner0) [reply index.html:62]: vvbukj/'l0[
\'
]/\

**Pull request merged**